npm/ng-packs/packages/components/extensible/src/lib/utils/typeahead.util.ts

```diff
@@ -39,7 +39,7 @@ export function createTypeaheadOptions(
 }
 
 export function getTypeaheadType(lookup: ExtensionPropertyUiLookupDto, name: string) {
-  if (!!lookup.url) {
+  if (!lookup.url) {
```

Comment:
This condition couldn't change

Reply:
Here is what I got for this condition, that is why I changed it in that way
<img width="258" alt="image" src="https://github.com/user-attachments/assets/14d72500-7eda-4482-ab2f-eae90988b6aa">